iamchrissmith/air_bnb_clone
# Create reservation

#### What does this PR do?
Adds the ability to create a reservation for a property via API.

Utilizes that API to submit the form on the Property page for a logged in user to make a reservation.
#### Migrations
NO  
#### Where should the reviewer start?
`views/properties/_booking.html.erb` and `assets/javascripts/reservations/submit.js`

#### How should this be manually tested?
1. Log in as a user
2. Visit a property page
3. Pick dates
4. Submit and you should see a confirmation message.

#### Any background context you want to provide?
Next steps:

- Validations on the form
- Display reservations in user dashboard

#### What are the relevant tickets?
[Finishes #149519739]
[#149519739](https://www.pivotaltracker.com/story/show/149519739)

#### Screenshots (if appropriate)
#### Questions:
  - Do Environment Variables need to be set?
  - Any other deploy steps?
